### PR TITLE
use a default pairing password Keycard

### LINKED ISSUE #12685: Make keycard pairing password a default one
To simplify user experience the idea of using a fixed, default pairing password when setting up Keycard with Status has come up. The idea is that this way we can do pairing without any user interaction, unless the user intentionally changed the pairing password (the option to do so will remain).

Full description here: https://hackmd.io/8g0MuXW2TSixVpJDEs5jdg

To be done: 
[ ] make generated pairing code a default one 
[ ] any place in the flows where the pairing code is asked to the user, replace this by automatic fullfilment of the pairing code

### PR DESCRIPTION
**Pull request by bitgamma**
Closes #12685 

The new behaviour is:

1. when creating a new Keycard account with empty/factory reset card, the pairing password will be KeycardDefaultPairing
2. when recovering a Keycard account or restoring pairing in any way, the KeycardDefaultPairing password will be automatically tested before prompting the user for a pairing password. If it succeeds, the user skips the pairing step automatically, otherwise proceeds as before.

### CODE REVIEW

File: src/status_im/keycard/common.cljs [hunk 1]
```diff
@@ -416,6 +417,16 @@
 
 ;; Get application info
 
+(fx/defn update-pairings
+  [{:keys [db] :as cofx} instance-uid pairing]
+  (let  [paired-on (utils.datetime/timestamp)
+         pairings  (-> (get-in db [:keycard :pairings])
+                       (assoc  instance-uid {:pairing pairing :paired-on paired-on}))]
+    (fx/merge
```
Comment (flexsurfer) [inline src/status_im/keycard/common.cljs]: merge is redundant here

Comment (flexsurfer) [reply src/status_im/keycard/common.cljs]: you can just return a map

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/keycard/common.cljs (hunk 1 above)
Comment (bitgamma) [reply src/status_im/keycard/common.cljs]: fixed, thanks!

**Comment by qoqobolo** [comment]
@bitgamma thanks for the PR!

### ISSUE 1: Onboarding flow gets stuck at the creating passcode step 
When trying to create/recover a Keycard account the onboarding process stops after confirming the passcode.

**_Steps:_**
1. Install Status
2. `Generate keys` > `Choose a name` > Choose Keycard > `Begin setup` 
3. Create and confirm a passcode

**OS:** Android, iOS

https://user-images.githubusercontent.com/67952253/141773252-13d08ef9-e938-4c6b-979f-aeb2ecadf371.MP4

**Comment by bitgamma** [comment]
@qoqobolo thanks for testing it! Sorry about that bug, somehow I hadn't seen it. It is fixed now!

**Comment by qoqobolo** [comment]
@bitgamma thank you! Everything is fine now. 
PR is tested and can be merged (failed e2e are not related to PR).

**Pull request merged**